c10/core/DispatchKey.cpp

```diff
@@ -63,30 +63,33 @@ const char* toString(DispatchKey t) {
     case DispatchKey::Meta:
       return "Meta";
 
+    case DispatchKey::Autograd:
+      return "Autograd";
+    case DispatchKey::AutogradCPU:
+      return "AutogradCPU";
+    case DispatchKey::AutogradCUDA:
+      return "AutogradCUDA";
+    case DispatchKey::AutogradXLA:
+      return "AutogradXLA";
+    case DispatchKey::PrivateUse1_PreAutograd:
```

Comment:
Probably need to rename these too at some point.

Reply:
Also, I remember you mentioning that we could have an arbitrary usage of `PrivateUse1_PreAutograd`, e.g., to intercept autograd for any dispatch key, not just `PrivateUse1`. But it may be easier to understand and simpler to require `PrivateUse1Autograd` be used for `PrivateUse1`.

Reply:
Yea was thinking about renaming these keys together with XLA but I'm not sure who'll be broken by it. Maybe I can open an issue for it to notify users and change it later?

Reply:
Sure. At the very least nested tensor will stop working. cc @cpuhrsch 

You could always add aliases for the legacy names at the bottom of DispatchKey to maintain BC too.